src/script/main/app.ts

```diff
@@ -415,15 +415,18 @@ export class App {
       onProgress(10);
       telemetry.timeStep(AppInitTimingsStep.INITIALIZED_CRYPTOGRAPHY);
 
-      const {members: teamMembers} = await teamRepository.initTeam(selfUser.teamId);
+      const connections = await connectionRepository.getConnections();
+
       telemetry.timeStep(AppInitTimingsStep.RECEIVED_USER_DATA);
 
-      const connections = await connectionRepository.getConnections();
       telemetry.addStatistic(AppInitStatisticsValue.CONNECTIONS, connections.length, 50);
 
       const conversations = await conversationRepository.loadConversations();
 
-      await userRepository.loadUsers(selfUser, connections, conversations, teamMembers);
+      const contacts = await userRepository.loadUsers(selfUser, connections, conversations);
+      if (selfUser.teamId) {
+        await teamRepository.initTeam(selfUser.teamId, contacts);
+      }
 
```

Comment:
We have to change the order at which things happened. 
Before, we loaded the team users and the connected users at the same time. 

Now the idea is to load all the users we interacted (or are connected) with first and then sort out which ones are team members later on